gr-uhd/examples/python/freq_hopping.py

```diff
@@ -109,16 +109,22 @@ def __init__(
         gain_tag.value = pmt.to_pmt({'gain': tx_gain})
         tag_list = [gain_tag,]
         for i in xrange(len(self.hop_sequence)):
+            time = pmt.cons(
+                    pmt.from_uint64(int(base_time + i * hop_time+0.01)),
+                    pmt.from_double((base_time + i * hop_time+0.01) % 1),
+            )
             tune_tag = gr.tag_t()
             tune_tag.offset = i * burst_length
             if i > 0 and post_tuning and not dsp_tuning: # TODO dsp_tuning should also be able to do post_tuning
                 tune_tag.offset -= 1 # Move it to last sample of previous burst
             if dsp_tuning:
                 tune_tag.key = pmt.string_to_symbol('tx_command')
                 tune_tag.value = pmt.to_pmt({'lo_freq': base_freq, 'dsp_freq': base_freq - self.hop_sequence[i]})
```

Comment:
Would look much nicer if the time gets rolled into the initial dictionary.

Reply:
I don't know yet how to do the squash without making another github pull request. I'll do that as soon as I learn that.

Regarding rolling it into initial dictionary - I did it this way because:
* the initial dictionary is a Python's dictionary that is converted into pmt object,
* I don't know how to confine types so after the conversion from python objects to pmt we get exactly pmt.cons with uint64 as car and double as cdr.

Therefore I preferred to create 'time' the pmt way. I can make this part more consistent by creating whole tune_tag.value in pmt way, instead of doing part by converting from Python and part using directly pmt.